cpp/src/decisiontree/batched-levelalgo/kernels.cuh

```diff
@@ -362,50 +367,132 @@ __global__ void computeSplitClassificationKernel(
     col = select(colIndex, treeid, node.info.unique_id, seed, input.N);
   }
 
-  for (IdxT i = threadIdx.x; i < len; i += blockDim.x) shist[i] = 0;
+  // populating shared memory with initial values
```

Comment:
Thanks for adding the comments. It helps me understand the code better.

Reply:
😄